server/controllers/work.js

```diff
@@ -15,3 +18,38 @@ export const work = async(ctx, next) => {
 
   return next();
 };
+
+function getResultsWithImages(results) {
+  if (!results) return;
+
+  return results.results.map((result) => {
+    const miroId = result.identifiers[0].value;
```

Comment:
Just an FYI: the identifiers structure will almost certainly change:

* We want to change the names of some of the fields
* There may be more than one identifier in the list

But that will probably happen when we change the image location, so this is fine for now.